public/app/features/explore/__snapshots__/Wrapper.test.tsx.snap

```diff
@@ -1790,6 +1790,13 @@ exports[`Wrapper changes the document title of the explore page 1`] = `
     
     .css-2p0j65-collapse__header-label{font-weight:500;margin-right:8px;font-size:14px;}
   </style>
+  <style
```

Comment:
@aocenas @ivanahuckova -- is this reasonable?  If yes, can you click the merge button on this PR?  (it is past my bed time!)

Reply:
honestly not sure why these changes would trigger difference in this snapshot. Don't think this is an issue could be because of some dependency update theoretically.

Reply:
Hmm sorry there seems to be some conflicts with this now. Maybe somebody else was already dealing with this.

Reply:
I just merged a main branch where this file already deleted: in this commit https://github.com/grafana/grafana/commit/3a68409e5d102ca77d3c2cbf4bcfea382d2bc826#diff-57c6144da8e092b945500b0c610fb633b6d1d6a04dba29cd068c865d85e2b75c